sql/connect/client/jvm/src/test/scala/org/apache/spark/sql/PlanGenerationTestSuite.scala

```diff
@@ -3193,6 +3193,66 @@ class PlanGenerationTestSuite
     binary.select(fn.hll_union_agg("bytes"))
   }
 
+  test("theta_sketch_agg with column lgNomEntries") {
```

Comment:
To be honest, this part adds a lot of JSON and .proto.bin file contents to this PR without a lot of test coverage benefit. I'd remove this part entirely and focus on golden file test coverage with .sql/.sql.out files, and regular unit test coverage.

Reply:
I can take them out and keep the sql files